### PR TITLE
Add election and political party models

### PR DESCRIPTION
**Pull request by yawboakye**
Resolves https://github.com/devcongress/elections-api/issues/2 and https://github.com/devcongress/elections-api/issues/3

### CODE REVIEW

File: test/controllers/v1/elections_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @election = elections(:one)
+  end
+
+  test "should find election by id" do
+    get v1_election_url(@election),
+        headers: {
+          "Accept": "application/vnd.api+json"
+        }
+
+    assert_equal "application/vnd.api+json", @response.content_type
```
Comment (silentworks) [inline test/controllers/v1/elections_controller_test.rb]: Not sure if this test is doing what it says? is this not missing an `assert_response :ok`

Comment (yawboakye) [reply test/controllers/v1/elections_controller_test.rb]: Missed it. I'll add it now. Thanks for the catch :+1:

**Comment by yawboakye** [comment]
@silentworks Take another look. I also modified it to set the response `Content-Type` as `application/json` when that was the request's `Accept` value. In the past it was sending `application/vnd.api+json` for all cases.

**Review by silentworks — APPROVED** [review]
This looks good to be merged in.

**Review by petersowah — APPROVED** [review]

**Comment by yawboakye** [comment]
@silentworks @petersowah I updated with new changes. If this is :+1: please go ahead with the merge. I worked on parameter parsing to support the `application/vnd.api+json` content type, and I think that should be there for everyone working on the API. Medaase!

**Pull request merged**